DESCRIPTION

```diff
@@ -28,11 +28,12 @@ Imports:
     stringr,
     tidyr,
     tlf
-Suggests: 
+Suggests:
+    ggplot2,
```

Comment:
@IndrajeetPatil @PavelBal @msevestre 
Do we want to have `tlf` and `patchwork` as **mandatory** prerequisites (like now) or is it sufficient to have them as **optional** prereqs? 
I would vote for **optional**, because - except for plots - `ospsuite` is still fully functional without those packages installed.

Reply:
Honestly speaking, I think this function should live in _tlf_. It sticks out like a sore thumb in _tlf_ at the moment. We can re-export it from _ospsuite_ for convenience, but it doesn't feel like its natural home.

In any case, I can move patchwork to `Suggests`.

Reply:
But I think this should be part of a separate PR. This is not in the scope of the current PR.

Reply:
>I think this function should live in tlf. 

Which function?

Reply:
`plotGrid()`

Reply:
It's a general plotting function with no ties to `DataCombined`, and so more appropriate for _tlf_ than _ospsuite_.

Reply:
ah ok. Yes, I agree. Would expect it in the tlf as well.

Reply:
Cool, I will create a separate issue to track this, both here and in tlf.